src/core/ActionWithVector.h

```diff
@@ -173,7 +173,7 @@ class ActionWithVector:
 /// This is to transfer data from the buffer to the final value
   void finishComputations( const std::vector<double>& buf );
 /// Apply the forces on this data
-  void apply() override;
+  virtual void apply() override;
```

Comment:
I think "virtual" is not needed here: usually you have either virtual, or override, or nothing